01_intro-r.qmd

```diff
@@ -508,32 +575,41 @@ This simplifies code and makes it portable because all relative file paths will
 
-## Twitter
+### Twitter
 
-Twitter is mostly bad. But the #rstats hashtag and #rstats community are mostly good. They are also generally inclusive and civil. In particular, open sources developers like Hadley Wickham (@hadleywickham), Jenny Bryan (@JennyBryan), and Joe Cheng (@jcheng) are active.
+Twitter is mostly bad. But the #rstats hashtag and #rstats community are mostly good. They are also generally inclusive and civil. In particular, open sources developers like Hadley Wickham (\@hadleywickham), Jenny Bryan (\@JennyBryan), and Joe Cheng (\@jcheng) are active.
 
-## Data Science for Public Policy Slack
+### Data Science for Public Policy Slack
 
 We've created a Slack workspace for this class (which will be shared across both sections) and encourage you to ask questions in Slack. In general, we ask that you try to answer questions on your own using the sources above *before* posting in Slack. Practicing finding and applying the relevant information to answer your questions is an important data science skill! The teaching staff will be checking the Slack to help answer questions in a reasonable time frame and we also encourage you to answer each other's questions - it's a great way to improve your R skills!
 
-Questions on Slack must be asked using [reproducible examples](http://adv-r.had.co.nz/Reproducibility.html). Simply copying-and-pasting questions or answers in the Slack channel is not allowed. If you're unsure how to share a reproducible example without sharing your answers in a public channel, you can DM the teaching staff to be safe. 
\ No newline at end of file
+Questions on Slack must be asked using [reproducible examples](http://adv-r.had.co.nz/Reproducibility.html). Simply copying-and-pasting questions or answers in the Slack channel is not allowed. If you're unsure how to share a reproducible example without sharing your answers in a public channel, you can DM the teaching staff to be safe. 
+
+### ChatGPT:
+
+Since there is R code on the internet, ChatGPT has been trained on R code and has the capability to answer R coding questions. Exercise extreme caution when using ChatGPT! ChatGPT saves and uses the queries you provide it. This means that asking a question about sensitive data or code could expose that data. If you decide to use ChatGPT, only ask queries of it using a [reproducible example](https://stackoverflow.com/questions/5963269/how-to-make-a-great-r-reproducible-example) with non-sensitive data. The diamonds dataset, loaded with `ggplot::diamonds()` is a great candidate. 
+
+This warning aside, ChatGPT can be a powerful tool. Some helpful tips for using ChatGPT for coding questions are:
+- Provide it detailed questions
+- Give it reproducible example code
+- Refine queries when the initial responses are unsatisfactory
```

Comment:
Maybe add the following:

Just like machine translators are useful for communication but can stymie the ability to learn a language like Spanish, large language models are useful for creating code but can stymie learning R. 

We encourage everyone to pursue mastery of R. This means feeling the terror of the blank page and internalizing the syntax of R without the crutch of LLMs before regularly using LLMs.

Reply:
Good notes! Added